integration_tests/steps/load_tests.py

```diff
@@ -51,6 +52,28 @@ def check_traded_energy_rate(context):
                        load.strategy.bid_update.final_rate[market.time_slot]
 
 
+@then('the Load strategy increases its accepted offers price as expected')
+def load_price_increase(context):
+    house = list(filter(lambda x: x.name == "House 1", context.simulation.area.children))[0]
+    load = list(filter(lambda x: "H1 General Load" in x.name, house.children))[0]
+
+    number_of_available_updates = \
+        int(load.config.slot_length.seconds / load.strategy.bid_update.update_interval.seconds) - 1
+    energy_rate_change_per_update = 4
+    rate_list = \
+        [ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE - i *
+         energy_rate_change_per_update
+         for i in range(number_of_available_updates)]
+    if load.strategy.bid_update.fit_to_limit is False:
+        for market in house.past_markets:
+            for trade in market.trades:
+                if trade.buyer == load.name:
+                    assert any([isclose(trade.offer.price / trade.offer.energy,
+                                        rate) for rate in rate_list])
```

Comment:
This does not check whether the offer price really got increased. There is only one trade in each market and probably they have all the same price. 
I think you can only check it with a unittest. Please correct me, if I overlook something. Thanks!